content/tips/livewire-disable-poll-when-hidden.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Livewire Disable Poll When Hidden'
+tweet_id: '1383079668358725636'
+thread_slug: ''
+author_username: abrardev99
+images:
+    - 'https://i.imgur.com/wyVHPhW.png'
```

Comment:
Could you remove the margin on this image?

Reply:
I updated the image.